crates/next-core/src/next_config.rs

```diff
@@ -1229,6 +1229,11 @@ impl NextConfig {
         Vc::cell(self.output == Some(OutputType::Standalone))
     }
 
+    #[turbo_tasks::function]
+    pub fn ci_has_next_support(&self) -> Vc<bool> {
+        Vc::cell(self.env.contains_key("NOW_BUILDER"))
```

Comment:
Also match heroku? https://github.com/vercel/next.js/blob/a8c9d07a026a02831af83d874a91f42dc58a8866/packages/next/src/server/ci-info.ts#L7-L9

Reply:
The heroku line only influences `isCI` and `name`, not `hasNextSupport`.